package.json

```diff
@@ -10,6 +10,7 @@
     "@svgr/webpack": "2.4.1",
     "@types/material-ui": "^0.21.6",
     "@types/react-router-dom": "^4.3.3",
+    "@types/request": "^2.48.1",
```

Comment:
You're right- I added this line locally because my machine was having problems finding the requests package, but I just removed it and everything is still working fine.